source/stable-release_iot/feature-delta-iot.rst

```diff
@@ -0,0 +1,76 @@
+.. _feature-delta-iot:
+
+Feature Delta Offered by Celadon IoT
```

Comment:
Hi @usivarax, I still see we are prominently using 'Celadon IoT' every where. There has been alignment on using 'Stable Releases (IoT), Would you please search replace 'Celadon IoT' with 'Stable Releases (IoT)'

Reply:
Updated as per the Suggestion